platform/android/scripts/validate-license.py

```diff
@@ -10,7 +10,7 @@
 ## Git diff changes
 p = Popen(['git', 'diff', '--name-only', 'LICENSE.md'], stdin=PIPE, stdout=PIPE, stderr=PIPE)
 output, err = p.communicate(b"input data that is passed to subprocess' stdin")
-if "platform/android/LICENSE.md" in output:
+if b"platform/android/LICENSE.md" in output:
```

Comment:
what is this additional `b` doing?

Reply:
If will show the following error on my laptop
```
Traceback (most recent call last):
  File "scripts/validate-license.py", line 13, in <module>
    if "platform/android/LICENSE.md" in output:
TypeError: a bytes-like object is required, not 'str'
```
Fix it according https://stackoverflow.com/a/42612820